saas/views/users.py

```diff
@@ -58,13 +58,19 @@ def get_template_names(self):
     def get_context_data(self, **kwargs):
         context = super(ProductListView, self).get_context_data(**kwargs)
         user = self.user
+        api_accessibles_url = reverse(
+            'saas_api_accessibles', args=(user,)
+        )
+
+        # Append the query parameter to include the personal profile
+        api_accessibles_url += '?include_personal_profile=true'
```

Comment:
It seems `?include_personal_profile=true` should be added only if passed to the `View` itself as a query parameter.